docs/guides/ingress/README.md

```diff
@@ -45,7 +45,7 @@ Guides show you how to use Voyager as a Kubernetes Ingress controller.
   - [Specify NodePort](/docs/guides/ingress/configuration/node-port.md)
   - [Configure global options](/docs/guides/ingress/configuration/default-options.md)
   - [Configure Custom Timeouts for HAProxy](/docs/guides/ingress/configuration/default-timeouts.md)
-  - [Configure Load balancing algorithm](/docs/guides/ingress/configuration/loadbalance-algorithm.md)
+  - [Configure Load balancing algorithm](/docs/guides/ingress/configuration/loadbalancing-algorithm.md)
```

Comment:
I've renamed the file as well. If you don't like renaming files (because they break incoming links), I can drop this commit...